### PR TITLE
supplemnt a mermaid graph for multi-zone

### LINKED ISSUE #1767: The multi-zone section lacks of a graph
### What happened?

https://kuma.io/docs/2.7.x/production/deployment/multi-zone/#how-it-works
WeChat7bb7101bfacb22d82140bda39dbcbb20

### PR DESCRIPTION
**Pull request by Icarus9913**
close #1767 

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) - yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? -yes

### CODE REVIEW

**Review by bartsmykla — CHANGES_REQUESTED** [review]
It doesn't work: https://deploy-preview-1771--kuma.netlify.app/docs/2.7.x/production/deployment/multi-zone/#how-it-works

File: app/_src/production/deployment/multi-zone.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ This way you can make a service multi-zone by having data planes using the same
...
+graph TD
+    subgraph zone-A
+        client[Client]
+        serviceA[Service Backend A]
+    end
+    subgraph zone-B
+        serviceB[Service Backend B]
+    end
+    client --> serviceB
+```
```
Comment (bartsmykla) [inline app/_src/production/deployment/multi-zone.md:41]:
```suggestion
{% mermaid %}
graph TD
    subgraph zone-A
        client[Client]
        serviceA[Service Backend A]
    end
    subgraph zone-B
        serviceB[Service Backend B]
    end
    client --> serviceB
{% endmermaid %}
```

File: app/docs/2.0.x/deployments/multi-zone.md [hunk 2]
```diff
@@ -23,6 +23,18 @@ This way you can make a service multi-zone by having data planes using the same
...
+graph TD
+    subgraph zone-A
+        client[Client]
+        serviceA[Service Backend A]
+    end
+    subgraph zone-B
+        serviceB[Service Backend B]
+    end
+    client --> serviceB
+```
```
Comment (bartsmykla) [inline app/docs/2.0.x/deployments/multi-zone.md:36]:
```suggestion
{% mermaid %}
graph TD
    subgraph zone-A
        client[Client]
        serviceA[Service Backend A]
    end
    subgraph zone-B
        serviceB[Service Backend B]
    end
    client --> serviceB
{% endmermaid %}
```

Comment (Icarus9913) [reply app/docs/2.0.x/deployments/multi-zone.md]: OK, got it Bart. Thank you.

I just changed it and force pushed to avoid multiple commits.

**Comment by Icarus9913** [comment]
The current implementation picture is not accurate, close it for now.

**Pull request closed without merging**